beacon_node/network/src/subnet_service/attestation_subnets.rs

```diff
@@ -145,31 +161,83 @@ impl<T: BeaconChainTypes> AttestationService<T> {
         }
     }
 
+    #[cfg(feature = "deterministic_long_lived_attnets")]
+    pub fn new(
+        beacon_chain: Arc<BeaconChain<T>>,
+        node_id: ethereum_types::U256,
+        config: &NetworkConfig,
+        log: &slog::Logger,
+    ) -> Self {
+        let log = log.new(o!("service" => "attestation_service"));
+
```

Comment:
Maybe we should an an INFO log saying something like:
"Deterministic subnets enabled; "subnets_per_node" => spec.subnets_per_node

Reply:
done in https://github.com/sigp/lighthouse/pull/3453/commits/97aa24aa4b3a41d1882cd407cb05dccce99774bc